src/include/packager/repositories_include.rb

```diff
@@ -497,5 +461,22 @@ def EditDialog
       ret = SourceDialogs.EditDialog
       ret
     end
+
+    # Evaluate the default autorefresh flag for the given repository URL.
+    # @param url [String] Repository URL
+    # @return [Boolean] The default autorefresh flag for the URL
+    def autorefresh(url)
+      log.info "Evaluating autorefresh flag for #{URL.HidePassword(url)}"
+      protocol = URL.Parse(url)["scheme"].downcase
+
+      # disable autorefresh for local repositories ,
+      # see https://github.com/openSUSE/libzypp/blob/master/zypp/Url.cc#L458
+      # for the local URL protocols defined by libzypp
+      local_protocols = [ "cd", "dvd", "dir", "hd", "iso", "file" ]
```

Comment:
and also used in other places like `add-on/src/clients/inst_add-on.rb:117` or here in `packager/src/clients/repositories.rb:327` or `packager/src/modules/SourceDialogs.rb:2094` 

Ah, and that constant actually already exists in yast2/library/packages/src/lib/packages/repository.rb:29 - `LOCAL_SCHEMES = [:cd, :dvd, :dir, :hd, :iso, :file].freeze`